velodrome/transform/fetcher.go

```diff
@@ -53,14 +52,14 @@ func fetchRecentIssues(db *gorm.DB, repository string, last *time.Time, out chan
 }
 
 // fetchRecentEventsAndComments retrieves events from DB, but only fetches events created since last call
-func fetchRecentEventsAndComments(db *gorm.DB, repository string, lastEvent *int, lastComment *int, out chan interface{}) error {
-	glog.Infof("Fetching issue-events with id bigger than %d", *lastEvent)
-	glog.Infof("Fetching comments with id bigger than %d", *lastComment)
+func fetchRecentEventsAndComments(db *gorm.DB, repository string, lastEvent *time.Time, lastComment *time.Time, out chan interface{}) error {
```

Comment:
Yeah, good point. Not sure it ever made sense like this, but it definites doesn't now.